### PR TITLE
feat: replace next-runtime-env with `t3/env`

### PR DESCRIPTION
**Pull request by G3root**

### CODE REVIEW

File: src/components/update/editor.tsx [hunk 1]
```diff
@@ -41,7 +41,7 @@ const UpdatesEditor = ({
 }: UpdatesEditorProps) => {
   const router = useRouter();
   const { toast } = useToast();
-  const baseUrl = env("NEXT_PUBLIC_BASE_URL");
+  const baseUrl = NEXT_PUBLIC_BASE_URL();
```
Comment (dahal) [inline src/components/update/editor.tsx:44]: Can change this to just `NEXT_PUBLIC_BASE_URL` instead of `NEXT_PUBLIC_BASE_URL()` @G3root

**Comment by G3root** [comment]
@dahal made some changes. now `PUBLIC_` envs can be accessed through `t3/env` and dependency `next-runtime-env` has been removed.

**Comment by G3root** [comment]
@dahal The page `/updates/new` throws a client exception because it already has an error while trying to access the document on the server. As a result, the env script isn't rendering.

**Comment by G3root** [comment]
@dahal the issue has been fixed , now it's ready for review

**Review by dahal — APPROVED** [review]
🚀

**Pull request merged**